python/Dockerfile

```diff
@@ -1,8 +1,10 @@
 FROM python:3.8
-ADD requirements.txt requirements.txt
+
+WORKDIR /app
+ADD . .
```

Comment:
Docker file will be in image, layers will be cached not exactly right

Reply:
The purpose of this to have html folder inside docker, since templates have symlinks. I think having Dockerfile inside is not a big deal.